Seed.Domain.Tests/Services/UserServiceTests.cs

```diff
@@ -1,71 +1,202 @@
 using Microsoft.EntityFrameworkCore;
+using Moq;
 using Seed.Data.EF;
 using Seed.Data.Models;
 using Seed.Domain.Services;
+using Seed.Domain.Services.Interfaces;
 using System;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace Seed.Domain.Tests
 {
     public class UserServiceTests
     {
+        #region Members
+
+        private readonly Mock<WebApiCoreSeedContext> _context;
+        private readonly UserService _userService;
+
+        #endregion
+
+        #region Constructors
+
+        public UserServiceTests()
+        {
+            _context = new Mock<WebApiCoreSeedContext>();
+            _userService = new UserService(_context.Object);
```

Comment:
For this particular class (the one under test) we can't do the same as their dependencies. We have to create a new instance per test. In that way, you ensure that the class under test is a pristine brand new instance. Otherwise, each test will modify the same instance and probably contaminate the results.